VideoLocker/res/layout/panel_custom_action_bar.xml

```diff
@@ -18,13 +18,13 @@
         android:layout_marginLeft="11dp"
         android:layout_marginStart="11dp"
         android:orientation="vertical"
+        android:contentDescription="@string/cancel_btn"
```

Comment:
In Android, the word "button" is only added if the UI element is a `Button` or a `subclass` of `Button`. But we can't label a view as a button, so, have to add the word button as a string.